assembl/tasks/mac.py

```diff
@@ -0,0 +1,106 @@
+import os
```

Comment:
Creating this Mac file was a choice taken by Raphael to separate the production file from developer file. Its purpose was to avoid the `if (running_locally())` and if env.mac in all functions. It is not complete. Are we still going to keep it ?

Reply:
I think so, yeah. There will be a duplication of logic, but with the right abstraction, can be used in both places equally well. We'll keep it.